flymake-languagetool.el

```diff
@@ -214,30 +222,38 @@ non-nil."
 (when flymake-languagetool-use-categories
   (flymake-languagetool--category-setup))
 
+;; Ignore some faces
+(defun flymake-languagetool-ignore-at-pos-p (pos faces-to-ignore)
+  (let ((x (get-text-property pos 'face)))
+	(if (listp x)
+		(cl-intersection faces-to-ignore x)
+	  (memq x faces-to-ignore))))
```

Comment:
1. Space/Tab issue
2. We might need to require `cl-lib` in the top level due of the function `cl-intersection`?
3. I think we can use `if-let` here?

Reply:
1. OK
2. Alright
3. This one though, becuase of the `listp` check, it makes it bit more awkward to be `if-let`, I am more inclined to keep it the way it is now.

Reply:
Sounds good to me, let's keep it this way then. 👍